homeassistant/components/glances/config_flow.py

```diff
@@ -72,11 +78,10 @@ async def async_step_user(
                 )
```

Comment:
I must say, that validate_input doesn't have a lot of lines, what do you think about moving the content to the place we call it and just directly catch the Glances exceptions instead of creating our own in between?

Reply:
Sure I can do this. But I thought of keeping it as it will also be used in the re-auth step which I plan on adding. Should we keep it or?

Reply:
Ah right. I'll leave it up to you then. In that case, if we remove the whole try except from that function, we can directly catch GlancesApiAuthorizationError and GlancesApiConnectionError without raising an error in between

Reply:
You have a point there. I will try this as it means we don't need exceptions.py anymore.